src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.Standalone.csproj

```diff
@@ -44,6 +44,4 @@
     <ProjectReference Include="..\Datadog.Trace\Datadog.Trace.csproj" />
   </ItemGroup>
 
-  <Import Project="..\Datadog.Trace.Ci.Shared\Datadog.Trace.Ci.Shared.projitems" Label="Shared" />
```

Comment:
Does this still build? The `Crank/Importer.cs` files uses the `Datadog.Trace.Ci` namespace so it may depend on some types from there

Reply:
The types in namespace `Datadog.Trace.Ci` now live in `Datadog.Trace.dll`, which is referenced above via `<ProjectReference>`. See https://github.com/DataDog/dd-trace-dotnet/tree/lpimentel/merge-projects/src/Datadog.Trace/Ci